capstone_project/src/main/webapp/scripts/maps_script.js

```diff
@@ -12,121 +12,64 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Toggle to hide news feed if it is already showing when the
- * button is pressed again.
- * @param {string} schoolName
- * @return {boolean}
+/** Creates a map centered at the geographic center of the US and
+ * adds it to the page.
  */
-function toggleToHideFeed(schoolName) {
-  const x = document.getElementById(schoolName);
-  if (x.innerHTML !== '') {
-    x.innerHTML = '';
-    return true;
-  }
-  return false;
-}
-
-/** Generates a hard-coded feed of news articles for UCI. */
-function showUCINews() { // eslint-disable-line no-unused-vars
-  if (toggleToHideFeed('uci-news')) {
-    return;
-  }
-
-  const articles =
-      [
-        'https://news.uci.edu/2020/07/07/uci-chancellor-emeritus-michael-v-drake-named-university-of-california-president/',
-        'https://news.uci.edu/2020/06/30/virtual-nurses-make-a-real-difference/',
-        'https://news.uci.edu/2020/06/18/uci-podcast-jessica-millward-on-the-meaning-and-importance-of-juneteenth/',
-      ];
+function createMap() { // eslint-disable-line no-unused-vars
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.828502, lng: -98.579512}, zoom: 4});
 
-  let i;
-  for (i = 0; i < articles.length; i++) {
-    createArticleEntry(articles[i], 'uci-news');
-  }
+  // Create a marker for each school and load these onto the map.
+  loadMarkersOntoMap(map);
 }
 
-/** Generates a hard-coded feed of news articles for UW. */
-function showUWNews() { // eslint-disable-line no-unused-vars
-  if (toggleToHideFeed('uw-news')) {
-    return;
-  }
-
-  const articles =
-      [
-        'https://www.washington.edu/news/2020/07/15/robotic-camera-backpack-for-insects/?utm_source=UW%20News&utm_medium=tile&utm_campaign=UW%20NEWS',
-        'https://www.washington.edu/news/2020/07/08/uw-school-of-oceanography-holds-no-1-global-ranking-more-than-two-dozen-areas-in-top-50/?utm_source=UW%20News&utm_medium=tile&utm_campaign=UW%20NEWS',
-        'https://www.washington.edu/news/2020/07/16/wsas-2020/?utm_source=UW%20News&utm_medium=tile&utm_campaign=UW%20NEWS',
-      ];
-
-  let i;
-  for (i = 0; i < articles.length; i++) {
-    createArticleEntry(articles[i], 'uw-news');
-  }
+/** Fetches, loads, and creates a marker/infowindow for each school,
+ * and places this marker on the map.
+ * @param {Object} map - A google maps Map object.
+ */
+function loadMarkersOntoMap(map) {
+  fetch('/school-data').then((response) => response.json()).then((schools) => {
+    let i;
```

Comment:
is i used outside of the for loop scope? If not, I recommend setting initializing it on the for loop